root/components/SubHeader.js

```diff
@@ -7,9 +7,9 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-import React from 'react';
+import * as React from 'react';
 
-const SubHeader = ({subHeading}: {subHeading: string}) => (
+const SubHeader = ({subHeading}: {subHeading: string | React.Node}) => (
```

Comment:
Nitpick, `string` is a subtype of `React.Node` so you can just put `React.Node` here. (There's the definition if you're curious: https://github.com/facebook/flow/blob/27630518e31ccecfac1466883a5736a70309b116/lib/react.js#L13)